lib/elixir/lib/version.ex

```diff
@@ -677,7 +677,13 @@ defimpl String.Chars, for: Version.Requirement do
 end
 
 defimpl Inspect, for: Version.Requirement do
-  def inspect(%Version.Requirement{source: source}, _opts) do
-    "#Version.Requirement<" <> source <> ">"
+  def inspect(%Version.Requirement{source: source}, opts) do
+    formatted =
+      ("\"" <> source <> "\"")
+      |> Inspect.Algebra.color(:string, opts)
+      |> Inspect.Algebra.format(:infinity)
+      |> IO.iodata_to_binary()
+
+    "#Version.Requirement<" <> formatted <> ">"
```

Comment:
You should not render inside inspect. You want more something like this (untested):

```suggestion
    inner = Inspect.Algebra.color("\"" <> source <> "\"", :string, opts)
    Inspect.Algebra.concat(["#Version.Requirement<", inner, ">"])
```

Reply:
Got you. Fixed